src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2109,7 +2118,7 @@ private MemberDeclarationSyntax ParseMemberDeclarationOrStatementCore(SyntaxKind
                     //            missing ';'
                     //
                     // Unless there modifiers or attributes are present this is more likely to be a method call than a method definition.
```

Comment:
>there  [](start = 30, length = 6)

Not part of the PR, but consider correcting the sentence, perhaps by removing "there".